src/Operators/spectralelement.jl

```diff
@@ -195,6 +197,19 @@ Base.@propagate_inbounds function copyto_slab!(out, bc, slabidx)
     return nothing
 end
 
+Base.@propagate_inbounds function copyto_slab!(
+    out::Field{T, <:Spaces.FiniteDifferenceSpace},
+    bc,
+    slabidx,
+) where {T}
+    space = axes(out)
+    rbc = resolve_operator(bc, slabidx)
+    @inbounds for ij in node_indices(space)
+        set_node!(space, out, ij, slabidx, get_node(space, rbc, ij, slabidx))
+    end
+    return nothing
+end
```

Comment:
```suggestion
```
This is already handled by the generic method above. You probably need to remove `QS` and `Nq` from that method (since they don't exist in a column), but those values aren't used so I'm not sure why they were extracted in the first place.